### DISCUSSION TITLE
Problem loading new text file into existing project

### DISCUSSION THREAD

**Discussion by Drajjan** [ccbogel/QualCoder#792] [General]
Hi!
I have a project with six transcribed interviews that I have already coded . Today I tried to att a seventh transcribed interview by inporting it. it is a .docx file. I got several error-messages I couldnt decode. after that I noticed that the file showed up in the project folder in the computers file-system but not when I go to manage files in the qualcodet software. and if I try to import again I get an errorcode that the file already is there. 
Have anyone stumbled on this problem before? I guess I can remove it from the folder but I’m guessing the importissues still would pop up. 
Is there for example a problem importing new files into a project that you have started coding on? or is it something with where the files need to be for them to be able to be imported?

**Comment by ccbogel** [comment]
Hi. Normally there is no problem. It would be helpful to see the error messages.
Try deleting the file from the project folder,  and deleting any record of it in the database, and then try importing again.

Copy the project folder to have a backup, then,
To delete a record in the database,  go to Reports > database queries 
Run this sql to find the record:
Select id, name from source 

Then, once you have identified which entry is the one to remove:
Delete from source where id=99

The example uses 99, but that will be a different number for you.

Alternative solutions:
1 copy the Word text into a plain text file and import that into QualCoder
2 create a new text entry within QualCoder (pencil icon in Manage files) and copy and paste the plain text into that.

**Comment by Drajjan** [comment]
Thank you for your valuable time and input! I will attach a pic of the error messages from the screen if it can be of any value for you.
best regards 
Daniel Westrell


Skickat från min iPhone
